### ISSUE TITLE
s/Rectangle/RectBox/g && s/Square/Rectangle/g && add Dice

### ISSUE THREAD

**Issue by jeyemwey** [jofftiquez/scope-of-works#4]
In my understanding of the English language:

A Square is a flat shape where all four sides are of the same length and meet at a 90deg angle.

A Rectangle to me is a four-cornered shape where the opposing sides have an equal length.

A RectBox to me (in the lack of a better word) is a Rectangle with a third dimension, which has its own size.

A Dice however, is a Square with a third dimension whereas the third dimension is as long as the other ones.

TLDR: I will fix that in a PR, hold on.

**Comment by jofftiquez** [comment]
This is quite useful if added. @jeyemwey could you kindly add the tests as well?

**Comment by jeyemwey** [comment]
Apparently, there is another definition: http://lasp.colorado.edu/~bagenal/MATH/math4.html

**Comment by jofftiquez** [comment]
Thanks great reference! @jeyemwey

**Comment by jofftiquez** [comment]
Big help @jeyemwey thanks!
